libs/components/datetime/testing/src/modules/timepicker/timepicker-selector-harness.ts

```diff
@@ -0,0 +1,94 @@
+import { ComponentHarness, HarnessPredicate } from '@angular/cdk/testing';
+
+import { SkyTimepickerSelectorColumnHarness } from './timepicker-selector-column-harness';
+import { SkyTimepickerSelectorHarnessFilters } from './timepicker-selector-harness-filters';
+
+/**
+ * Harness for interacting with a timepicker selector in tests.
+ */
+export class SkyTimepickerSelectorHarness extends ComponentHarness {
+  /**
+   * @internal
+   */
+  public static hostSelector = '.sky-timepicker-container';
+
+  #getHours = this.locatorFor(
+    SkyTimepickerSelectorColumnHarness.with({
+      selector: '.sky-timepicker-column-hours',
+    }),
+  );
+  #getMinutes = this.locatorFor(
+    SkyTimepickerSelectorColumnHarness.with({
+      selector: '.sky-timepicker-column-minutes',
+    }),
+  );
+  #getMeridies = this.locatorForOptional(
+    SkyTimepickerSelectorColumnHarness.with({
+      selector: '.sky-timepicker-column-meridies',
+    }),
+  );
+
+  /**
+   * @internal
+   */
+  public static with(
+    filters: SkyTimepickerSelectorHarnessFilters,
+  ): HarnessPredicate<SkyTimepickerSelectorHarness> {
+    return new HarnessPredicate(SkyTimepickerSelectorHarness, filters);
+  }
+
+  /**
+   * Clicks the specified hour button, or throws an error if it does not exist.
+   */
+  public async clickHour(value: string): Promise<void> {
+    try {
+      await (await this.#getHours()).clickButton(value);
+    } catch {
+      throw new Error(`Unable to find hour button with label "${value}".`);
+    }
+  }
+
+  /**
+   * Clicks the specified minute button, or throws an error if it does not exist.
+   */
+  public async clickMinute(value: string): Promise<void> {
```

Comment:
should below the `clickMeridie` function alphabetically

Reply:
moved